### ISSUE TITLE
Not working PowerPoint.createPresentation on PowerPoint online

### ISSUE THREAD

**Issue by hirossyi73** [OfficeDev/office-js#432]
## Expected Behavior
If we execute "PowerPoint.createPresentation(base64)", we can create New PowerPoint file both Client PowerPoint and PowerPoint Online.

## Current Behavior
If we execute "PowerPoint.createPresentation(base64)" on PowerPoint Online, not working.
I checked reference, but don't write that this function only works Client PowerPoint.
https://docs.microsoft.com/en-us/javascript/api/powerpoint?view=office-js#powerpoint-createpresentation

## Steps to Reproduce, or Live Example
This is Sample code.

~~~ html
<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />

    <script src="Scripts/jquery-1.9.1.js" type="text/javascript"></script>
    <script src="Scripts/FabricUI/MessageBanner.js" type="text/javascript"></script>
    <script src="https://appsforoffice.microsoft.com/lib/1/hosted/office.js" type="text/javascript"></script>
    <!-- <script src="Scripts/Office/MicrosoftAjax.js" type="text/javascript"></script>  -->
    <!-- <script src="Scripts/Office/1/office.js" type="text/javascript"></script>  -->

    <link href="Home.css" rel="stylesheet" type="text/css" />
    <script src="Home.js" type="text/javascript"></script>

    <link rel="stylesheet" href="https://appsforoffice.microsoft.com/fabric/2.1.0/fabric.min.css">
    <link rel="stylesheet" href="https://appsforoffice.microsoft.com/fabric/2.1.0/fabric.components.min.css">
    
    <!-- link rel="stylesheet" href="Content/fabric.min.css" -->
    <!-- link rel="stylesheet" href="Content/fabric.components.min.css" -->
</head>
<body>
    <div id="content-main">
        <div class="padding">
            <button class="ms-Button ms-Button--primary" id="create-new-presentation">
                <span class="ms-Button-icon"><i class="ms-Icon ms-Icon--plus"></i></span>
                <span class="ms-Button-label">Create PowerPoint</span>
                <span class="ms-Button-description">Create PowerPoint</span>
            </button>
        </div>
    </div>

    <div class="ms-MessageBanner" id="notification-popup">
        <div class="ms-MessageBanner-content">
            <div class="ms-MessageBanner-text">
                <div class="ms-MessageBanner-clipper">
                    <div class="ms-font-m-plus ms-fontWeight-semibold" id="notification-header"></div>
                    <div class="ms-font-m ms-fontWeight-semilight" id="notification-body"></div>
                </div>
            </div>
            <button class="ms-MessageBanner-expand" style="display:none"><i class="ms-Icon ms-Icon--chevronsDown"></i> </button>
            <div class="ms-MessageBanner-action"></div>
        </div>
        <button class="ms-MessageBanner-close"> <i class="ms-Icon ms-Icon--x"></i> </button>
    </div>
</body>
</html>
~~~

~~~ javascript
(function () {
    "use strict";

    var messageBanner;

    Office.initialize = function (reason) {
        $(document).ready(function () {
            var element = document.querySelector('.ms-MessageBanner');
            messageBanner = new fabric.MessageBanner(element);
            messageBanner.hideBanner();

            $('#create-new-presentation').click(createNewPresentation);
        });
    };


    function createNewPresentation() {
        PowerPoint.createPresentation();
    }

    function getDataFromSelection() {
        Office.context.document.getSelectedDataAsync(Office.CoercionType.Text,
            function (result) {
                if (result.status === Office.AsyncResultStatus.Succeeded) {
                    showNotification('Selected text:', '"' + result.value + '"');
                } else {
                    showNotification('Error:', result.error.message);
                }
            }
        );
    }

    function showNotification(header, content) {
        $("#notification-header").text(header);
        $("#notification-body").text(content);
        messageBanner.showBanner();
        messageBanner.toggleExpansion();
    }
})();
~~~

## Context
I want to create new document on PowerPoint Online, but i can't.

## Your Environment
* Platform [Office Online]: ______
* Host [PowerPoint]: ______
* Office version number: Office Online
* Operating System: Windows 10
* Browser (if using Office Online): IE, Chrome etc

## Useful logs
- [○] Console Errors
- [○] Screenshots
![image](https://user-images.githubusercontent.com/8457265/54509799-1f404480-498e-11e9-884f-a6f8f258d105.png)

powerpoint-web-16.00.js:24 Uncaught (in promise) RichApi.Error: このブラウザーでは、要求された API はサポートされません。
    at new c (https://appsforoffice.microsoft.com/lib/1/hosted/powerpoint-web-16.00.js:24:267637)
    at b (https://appsforoffice.microsoft.com/lib/1/hosted/powerpoint-web-16.00.js:24:398915)
    at Object.OSF.DDA.issueAsyncResult (https://appsforoffice.microsoft.com/lib/1/hosted/powerpoint-web-16.00.js:24:60089)
    at Object.onComplete (https://appsforoffice.microsoft.com/lib/1/hosted/powerpoint-web-16.00.js:24:72267)
    at Object.callback (https://appsforoffice.microsoft.com/lib/1/hosted/powerpoint-web-16.00.js:24:114175)
    at z (https://appsforoffice.microsoft.com/lib/1/hosted/powerpoint-web-16.00.js:24:103227)

**Comment by JuaneloJuanelo** [comment]
thanks for reporting this. Is the file you are trying to create a PPTX? how large it is?

**Comment by hirossyi73** [comment]
I want to try creating a new(empty) PPTX file.

~~~
PowerPoint.createPresentation(); // not contains base64 args. So it creates new PPTX file.
~~~

And I tried to create a saved pptx file using base64.

~~~
PowerPoint.createPresentation('UEsDBBQABgAIAAAAIQDfzBj1r.......');
~~~

This file is very simply, not large. It's 34KB.
![image](https://user-images.githubusercontent.com/8457265/54572451-e86c3c00-4a2a-11e9-8d31-2036d27a67ad.png)

**Comment by jipyua** [comment]
The API is back and it's working now, please have a try.

**Comment by hirossyi73** [comment]
OK! I can create on PowerPoint Online!! Thanks!!
